guides/source/working_with_javascript_in_rails.md

```diff
@@ -141,6 +141,7 @@ follow this pattern.
 Built-in Helpers
 ----------------------
 
+If you are using Rails with the [Asset Pipeline](asset_pipeline.html) enabled, 
```

Comment:
I could change it to 'Unless you have disabled the asset pipeline"?

> it's possible to turn it off, and if you turn it off, it'll stop working

Of course. But I think the real issue is that devs may not realize that this functionality is part of the asset pipeline. I see this misconception a lot, hence the idea for a warning.